src/pages/redirect-page.tsx

```diff
@@ -1,17 +1,28 @@
-import React, { useCallback, useContext, useEffect, useMemo, useState, type ReactElement } from 'react'
-import Header from '../components/Header.jsx'
-import { ConfigProvider } from '../context/config-context.jsx'
-import { ServiceWorkerContext, ServiceWorkerProvider } from '../context/service-worker-context.jsx'
-import { setConfig, type ConfigDb } from '../lib/config-db.js'
-import { getSubdomainParts } from '../lib/get-subdomain-parts.js'
-import { isConfigPage } from '../lib/is-config-page.js'
-import { getUiComponentLogger, uiLogger } from '../lib/logger.js'
-import { tellSwToReloadConfig } from '../lib/sw-comms.js'
-import { translateIpfsRedirectUrl } from '../lib/translate-ipfs-redirect-url.js'
-import './default-page-styles.css'
-
-const uiComponentLogger = getUiComponentLogger('redirect-page')
-const log = uiLogger.forComponent('redirect-page')
+import React, {
+  useCallback,
+  useContext,
+  useEffect,
+  useMemo,
+  useState,
+  type ReactElement,
+} from "react"
+import Header from "../components/Header.jsx"
+import LoadingIndicator from "../components/loading-indicator.jsx"
+import { ConfigProvider } from "../context/config-context.jsx"
+import {
+  ServiceWorkerContext,
+  ServiceWorkerProvider,
+} from "../context/service-worker-context.jsx"
+import { setConfig, type ConfigDb } from "../lib/config-db.js"
+import { getSubdomainParts } from "../lib/get-subdomain-parts.js"
+import { isConfigPage } from "../lib/is-config-page.js"
+import { getUiComponentLogger, uiLogger } from "../lib/logger.js"
+import { tellSwToReloadConfig } from "../lib/sw-comms.js"
+import { translateIpfsRedirectUrl } from "../lib/translate-ipfs-redirect-url.js"
+import "./default-page-styles.css"
+
+const uiComponentLogger = getUiComponentLogger("redirect-page")
+const log = uiLogger.forComponent("redirect-page")
```

Comment:
can you please be sure to make only changes where there are new changes, and follow existing code style?